recipes/tcl/8.6.9/conanfile.py

```diff
@@ -0,0 +1,222 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanExceptionInUserConanfileMethod
+from conans.util.env_reader import get_env
+import os
+import shutil
+import tempfile
+
+
+class TclConan(ConanFile):
+    name = "tcl"
+    description = "Tcl is a very powerful but easy to learn dynamic programming language."
+    topics = ("conan", "tcl", "scripting", "programming")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://tcl.tk"
+    license = "TCL"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "fPIC": [True, False],
+        "shared": [True, False]
+    }
+    default_options = {
+        "fPIC": True,
+        "shared": False,
+    }
+    _source_subfolder = "sources"
+    requires = ("zlib/1.2.11")
+
+    @property
+    def _is_mingw_windows(self):
+        return self.settings.os == "Windows" and self.settings.compiler == "gcc"
+
+    @property
+    def _install_folder(self):
+        # conan-center-index forbids having share/man folders at the package root
```

Comment:
For the autotools installation, you can pass the `"datadir={}".format(os.path.join(self.package_folder, "bin", "share")` argument to the configure script. (It is automatically set to `{prefix}/share` in the configure script). I haven't verified it.

Does the nmake scripts also copy the scripts to a `share` subfolder?

Reply:
I have set the `prefix` to `install_dir`, so it should do what's expected.

I am currently installing VS 2015 to build locally, I'll tell you tomorrow (hopefully :D)